extras/basic_example/public/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <title>Licode Basic Example</title>
-    <script type="text/javascript" src="erizo.js"></script>
+    <script type="text/javascript" src="http://192.168.33.99:8080/erizo.js"></script>
```

Comment:
That's the IP that we use for the webrtc vagrant.  It's hard-coded in vagrantfile so I think it's OK to hard-code here.